Add a note about Idp connectivity (#1329)
neo4j/docs-operations: modules/ROOT/pages/authentication-authorization/sso-integration.adoc

dbms.security.authorization_providers=oidc-newsso,oidc-oldsso,native
----
======
. Check connectivity.
Neo4j needs to connect to the identity provider to discover settings and fetch public keys to verify tokens.

> Check firewall settings, security controls, and, if necessary, logs to ensure that the Neo4j server is able to connect to the identity provider using HTTPS.

Reviewer: ```suggestion
Check firewall settings and security controls, and, if necessary, logs to ensure that the Neo4j server is able to connect to the identity provider using HTTPS.
```

dbms.security.authorization_providers=oidc-newsso,oidc-oldsso,native
----
======

> . Check Connectivity. Neo4j will need to connect to the identity provider to discover settings and fetch public keys to verify tokens. Check firewall settings and security controls to ensure that the Neo4j server is able to connect to the identity provider using https.

Reviewer 2: ```suggestion
. Check Connectivity. Neo4j will need to connect to the identity provider to discover settings and fetch public keys to verify tokens. Check firewall settings, security controls and if necessary logs, to ensure that the Neo4j server is able to connect to the identity provider using https.
```

> . Check Connectivity. Neo4j will need to connect to the identity provider to discover settings and fetch public keys to verify tokens. Check firewall settings, security controls and if necessary logs, to ensure that the Neo4j server is able to connect to the identity provider using https.

Reviewer 3: ```suggestion
. Check connectivity.
Neo4j needs to connect to the identity provider to discover settings and fetch public keys to verify tokens.
Check firewall settings, security controls, and, if necessary, logs to ensure that the Neo4j server is able to connect to the identity provider using HTTPS.
```
